prism/src/main/java/org/apache/falcon/service/EntitySLAMonitoringService.java

```diff
@@ -436,19 +417,20 @@ private boolean checkEntityInstanceAvailability(String entityName, String cluste
         Entity entity = EntityUtil.getEntity(entityType, entityName);
         authenticateUser();
         try {
-            if (entityType.equals(EntityType.PROCESS.toString())){
+            if (entityType.equalsIgnoreCase(EntityType.PROCESS.toString())){
                 LOG.trace("Checking instance availability status for entity:{}, cluster:{}, "
                         + "instanceTime:{}", entity.getName(), clusterName, nominalTime, entityType);
                 AbstractWorkflowEngine wfEngine = WorkflowEngineFactory.getWorkflowEngine();
                 InstancesResult instancesResult = wfEngine.getStatus(entity, nominalTime, nominalTime, null, null);
-                if (instancesResult.getStatus().equals(APIResult.Status.SUCCEEDED)){
+                if (instancesResult.getInstances().length > 0
+                        && instancesResult.getInstances()[0].status.equals(InstancesResult.WorkflowStatus.SUCCEEDED)){
```

Comment:
User could have ignored the instance for some faulty code or input, which the user might rerun later. Not taking it in the pending instances could mean user might lost the track of the instance.